Automattic/wp-calypso
# (e2e)(chore) Fix failing tests in `authentication/authentication__google.ts`

## Proposed Changes

- [x] Fix the `Enter 2FA challenge if required - Challenge 2` example, which was failing because Google seems to have changed the flow for the login with Google dialog, adding a final step that wasn't present before.

## Testing Instructions

Run the `Authentication E2E Tests` build in TC from this branch, all tests from `authentication/authentication__google.ts` should pass.
